internal/tofu/transform_provider.go

```diff
@@ -161,6 +161,52 @@ func (t *ProviderTransformer) Transform(g *Graph) error {
 			// Direct references need the provider configured as well as initialized
 			needConfigured[absPc.String()] = absPc
 		}
+		// Provider function references
```

Comment:
I think it would be good to expand this comment to explain that resources/datasources are referencers, and they might use functions in their attributes, and this will then make sure we have edges between the resource/datasource and the provider whose function they use.